mail/signing-party/Portfile

```diff
@@ -33,7 +33,7 @@ use_configure       no
 depends_build       port:automake \
                     port:autoconf
 
-perl5.major         5.30
+perl5.major         5.34
```

Comment:
Most ports using the perl5 portgroup and a single version of perl are setting `perl5.branches`, which is why I missed this one. Setting `perl5.major` for a single value makes more sense to me though. We should probably standardise on one or the other?

Reply:
I'm indifferent. I happily leave it to others to decide what looks better and then we can unify it in other ports.

I would also prefer to standardise the syntax, for example in other ports using
```
set perl_branch     5.34
```
and alike. If we do that, we could migrate to a newer version much faster and more easily in the future.

I tried to do that to some ports in the past, but then some maintainers complained that including the whole portgroup was an overkill when all they needed was a single simple variable.

Reply:
Just using the same variable name would helpful, even without using the portgroup.